src/types.ts

```diff
@@ -306,7 +306,7 @@ export interface GetTokenTransfersReply {
     transfers: TokenTransfer[];
     syncStatus?: SyncStatus;
 }
-export interface GetTransfersRequest {
+export interface GetTokenTransfersRequest {
```

Comment:
`GetTransfersRequest` is used for the `getTokenTransfers` and for the `getNftTransfers`. Therefore, the interface has a name without specifics.